Projects/hCoreUI/Sources/hButton/hButton.swift

```diff
@@ -13,10 +13,9 @@ struct LargeButtonModifier: ViewModifier {
 struct MediumButtonModifier: ViewModifier {
     func body(content: Content) -> some View {
         content
+            .frame(minHeight: 40)
+            .frame(maxWidth: .infinity)
             .padding(.horizontal, 16)
-            .padding(.vertical, 8)
```

Comment:
I think we should have this. if for example content inside of a button has a height of 50, we will have no top and bottom spacing.

Reply:
right. will add back 👍